### ISSUE TITLE
JAWS doesn't read the label for form fields when used in live region

### ISSUE THREAD

**Issue by jnurthen** [FreedomScientific/VFO-standards-support#26]
### Summary

When I update the value of a form field in a live region JAWS does not read out the label of the form field. Example
https://codepen.io/anon/pen/aVRyxw
Press the "Trigger Update" button 

### Expected result

I would expect JAWS to read the new value and the label for the form field

### Actual result

JAWS just reads the new value

### Example

https://codepen.io/anon/pen/aVRyxw
Press the "Trigger Update" button 

### Additional Information

NVDA & Voiceover (Mac) both read the label as well as the value.

#### JAWS version and build number

2018 - 1710.42
 
#### Operating System and version

Windows 8.1

#### Browser and version:

Chrome 62

**Comment by stevefaulkner** [comment]
Thanks @jnurthen have confirmed the behavior in chrome.

**Comment by stevefaulkner** [comment]
@jnurthen  thanks for reporting, internal Bug 98026 - JAWS doesn't read the label for form fields when used in live region

**Comment by ggordon-vfo** [comment]
Fixed in JAWS 2018 in January 2018 update . Fix extends to all versions of JAWS 2019.
